<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillListFrame</class>
 <widget class="QFrame" name="SkillListFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>1058</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Skill List Language</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="LanguageBox">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>6</number>
       </property>
       <item>
        <property name="text">
         <string>German</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>French</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Italian</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Japanese</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Korean</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spanish</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UK</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>List Item Height: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ItemSizeSpinBox">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QListWidget" name="SkillListPreview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>256</width>
         <height>256</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Skill List Preview</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_editor">
       <item>
        <widget class="QTextEdit" name="SkillListEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::LineWrapMode::NoWrap</enum>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>320</height>
          </size>
         </property>
         <widget class="QPushButton" name="BtnL1">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>60</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ｌ１</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnL2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ｌ２</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnR1">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>60</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ｒ１</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnR2">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>20</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ｒ２</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnLeft">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>←</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnUp">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>120</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>↑</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnDown">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>200</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>↓</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnRight">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>160</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>→</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnSquare">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>160</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>□</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnX">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>200</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>×</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnCircle">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>160</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>○</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnTriangle">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>120</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>△</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnL3">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>210</y>
            <width>51</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ｌ３</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnR3">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>210</y>
            <width>51</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ｒ３</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnPlus">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>270</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>＋</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnSlash">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>270</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>／</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnParen1">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>270</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>（</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnParen2">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>270</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>）</string>
          </property>
         </widget>
         <widget class="QPushButton" name="BtnRepeat">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>270</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>＊</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Skill List Editor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LanguageBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ChangeLanguage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ItemSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ListItemHeightChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1184</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>1184</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SkillListEdit</sender>
   <signal>textChanged()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>SkillListTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnR2</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>769</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>759</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnR1</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>811</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>801</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnCircle</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>906</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>901</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnTriangle</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>870</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>863</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnX</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>952</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>940</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnSquare</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>906</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>927</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnParen2</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>1017</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>983</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnParen1</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>1024</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>1021</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnRepeat</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>1013</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>1052</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnSlash</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>1020</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>1060</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnR3</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>963</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>1057</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnL3</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>966</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>1053</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnPlus</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>1022</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>1057</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnDown</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>963</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>1055</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnRight</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>916</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>1016</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnLeft</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>920</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>979</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnUp</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>871</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>880</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnL1</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>820</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>827</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnL2</sender>
   <signal>clicked()</signal>
   <receiver>SkillListFrame</receiver>
   <slot>ButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>771</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>781</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ChangeLanguage(int)</slot>
  <slot>SkillListTextChanged()</slot>
  <slot>ListItemHeightChanged(int)</slot>
  <slot>ButtonPressed()</slot>
 </slots>
</ui>
